src/pages/index.js

```diff
@@ -0,0 +1,251 @@
+  const cardId = data._id;
+
+  cardNameEl.textContent = data.name;
+  cardImageEL.setAttribute("src", data.link);
+  cardImageEL.setAttribute("alt", data.name);
+
+  if(data.isLiked){
+    cardLikeBtn.classList.add("card__like-button_liked")
+  }
+
+  cardLikeBtn.addEventListener("click", () => {
+    if (cardLikeBtn.classList.contains("card__like-button_liked")){
+      api.dislikeCardInfo(cardId)
+      .then(() => {
+        cardLikeBtn.classList.toggle("card__like-button_liked");})
+        .catch(console.error);
+    }
+    else {
+      api.likeCardInfo(cardId)
+       .then(() => {
+        cardLikeBtn.classList.toggle("card__like-button_liked");})
+        .catch(console.error);
+    }
+  });
+
+  cardDeletebtn.addEventListener("click", () => {
+    cardToDelete = {
+      element: cardElement,
+      id: cardId
+    };
+    openModal(cardDeleteModal);
+  });
+
+  cardImageEL.addEventListener("click", () => {
+    openModal(previewModal);
+    previewModalImageEl.src = data.link;
+    previewModalImageEl.alt = data.name;
+    previewModalCaptionEl.textContent = data.name;
+  });
+
+  return cardElement;
+}
+
+function openModal(modal) {
+  modal.classList.add("modal_opened");
+  document.addEventListener("keyup", handleEscape);
+}
+
+modals.forEach((modal) => {
+  modal.addEventListener("click", (evt) => {
+    if (
+      evt.target.classList.contains("modal") ||
+      evt.target.classList.contains("modal__close-button")
+    ) {
+      closeModal(modal);
+    }
+  });
+});
+
+function closeModal(modal) {
+  modal.classList.remove("modal_opened");
+  document.removeEventListener("keyup", handleEscape);
+}
+
+function handleEditFormSubmit(evt) {
+  evt.preventDefault();
+  handleLoadingState(editModalSubmit, true);
+  api.editUserInfo({
+    name:  editModalNameInput.value,
+    about: editModalDescriptionInput.value
+  })
+    .then((data) =>{
+  profileName.textContent = data.name;
+  profileDescription.textContent = data.about;
+  closeModal(editModal);
+    })
+    .catch(console.error)
+    .finally(() =>{
+      handleLoadingState(editModalSubmit, false)}
+    )};
+
+function handleAddFormSubmit(evt) {
+  evt.preventDefault();
+  handleLoadingState(addCardSubmit, true);
+  api.createCardInfo({
+    link: addModalLinkInput.value ,
+    name: addModalNameInput.value
+  })
+   .then((data) =>{
+    const cardElement = getCardElement(data);
+    cardsList.prepend(cardElement);
+    addFormElement.reset();
+    disableButton(addCardSubmit);
+    closeModal(addCardModal);
+  })
+  .finally(() =>{
+  handleLoadingState(addCardSubmit, false)
+});
+}
+
+function handleAvatarFormSubmit(evt){
+  evt.preventDefault();
+  handleLoadingState(avatarCardSubmit, true);
+  api.editAvatarInfo(avatarModalInput.value)
+  .then((avatar) => {
+    avatarPicture.src = avatar.link;
+    closeModal(avatarModal);
+  })
+  .finally(() =>{
+  handleLoadingState(avatarCardSubmit, false)});
+}
```

Comment:
* You need to **format** the markup (and code) to show **indentation** and to make the code more readable. You can do that with `Prettier` or a combination: press `SHIFT + ALT + F`